src/Kernel32.Shared/Win32Exception.cs

```diff
@@ -46,7 +46,7 @@ public Win32Exception()
         /// <param name="error">The Win32 error code associated with this exception.</param>
         public Win32Exception(Win32ErrorCode error)
 #if DESKTOP
-            : base((int)error)
+            : this(error, error.GetMessage() ?? $"Unknown error (0x{(int)error:x8})")
```

Comment:
After some cleaning, rebuilding and also using the build.ps1 -restore I was able to run all the tests on VS 2015 on my machine with the same result, so it seems not to be something bound only to Console Prompt windows. What should we do about it?